.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,66 @@
+---
+name: Docker
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+  # Run tests for any PRs.
+  pull_request:
+    branches:
+      - master
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  # Run tests.
+  # See also https://docs.docker.com/docker-hub/builds/automated-testing/
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run tests
+        run: docker build . --file Dockerfile
```

Comment:
I'm a little confused as to why this job is called `test`. Wouldn't something like `build` be more appropriate or am I missing something here? :thinking:

Reply:
Yeah it makes sense, in a copy-pasted version it ran some tests as well - will change!